npc/quests/skills/merchant_skills.txt

```diff
@@ -65,7 +65,7 @@ alberta,83,96,5	script	Necko	4W_M_02,7,7,{
 			mes "Ppyakikakikakakakakakakaka!!";
 			close;
 		}
-		else if (JobLevel < 15) {
+		else if (BaseJob == Job_Merchant && JobLevel < 15) {
```

Comment:
The above if loop already checks for BaseClass == Job_Merchant, so I guess this is not necessary.